app/assets/javascripts/i18n.js

```diff
@@ -598,6 +598,8 @@
 
     if (typeof(translation) === "string") {
       translation = this.interpolate(translation, options);
+    } else if (isArray(translation)) {
+      translation = translation.map((t) => this.interpolate(t, options));
```

Comment:
1. This `() => something` syntax is not ES5 syntax, please update it
2. Please add shim for `Array#map` to `app/assets/javascripts/i18n/shims.js`